### ISSUE TITLE
Custom dropout

### ISSUE THREAD

**Issue by sidkk86** [keras-team/keras#9517]
Hello,

I am trying to write a custom layer which drops nodes with probability p but in the inverted dropout scales the output by some other factor ( say p1). Keras seems to be calling tensorflow's standard dropout implementation which rescales by a factor of p. How should I write a layer which does what I want?

**Comment by sidkk86** [comment]
I had simply made a mistake in my code. Sorry for the inconvenience
